api/prisma/oldschema.prisma

```diff
@@ -198,6 +198,7 @@ model user {
   verificationToken            String? // Undefined
   website                      String? // Undefined
   yearsTopContributor          String[]             @default([]) // Undefined | String[]
+  isClassroomAccount           Boolean? // Undefined
```

Comment:
I was not sure if I should add this to the old schema as well.

Reply:
I don't see a probably with adding this.